src/main/java/org/fundacionjala/coding/jorge/FizzBuzz.java

```diff
@@ -0,0 +1,45 @@
+package org.fundacionjala.coding.jorge;
+
+import java.util.stream.IntStream;
+
+/**
+ *
+ */
+public final class FizzBuzz {
+    private static final int NUMBER_5 = 5;
+    private static final int NUMBER_3 = 3;
+    private static final int NUMBER_10 = 10;
+
+    /**
+     *
+     */
+    public FizzBuzz() {
+    }
+
+    /**
+     * @param rango int[]
+     * @return Object[]
+     */
+    public Object[] fizzBuzz(final int[] rango) {
+        return IntStream.rangeClosed(rango[0], rango[1])
+       .mapToObj(i -> i % NUMBER_5 == 0 ? i % NUMBER_3 == 0 ? "FizzBuzz" : "Buzz" : i % NUMBER_3 == 0 ? "Fizz" : i)
+                .toArray();
+    }
+
+    /**
+     * The Two Parts.
+     *
+     * @param rango int[].
+     * @return Object[].
+     */
+    public Object[] fizzBuzz2(final int[] rango) {
+        return IntStream.rangeClosed(rango[0], rango[1])
+                .mapToObj(i -> i % NUMBER_5 == 0 ? i % NUMBER_3 == 0 ? "FizzBuzz" : "Buzz"
+                        : i % NUMBER_10 == NUMBER_5 ? "Buzz"
+                        : (i / NUMBER_10) % NUMBER_10 == NUMBER_5 ? "Buzz"
+                        : i % NUMBER_10 == NUMBER_3 ? "Fizz"
+                        : (i / NUMBER_10) % NUMBER_10 == NUMBER_3 ? "Fizz"
+                        : i % NUMBER_3 == 0 ? "Fizz"
+                        : i).toArray();
```

Comment:
@AT-08/at-08 
I recommend you to have two methods 
* one of them given a single number -> return/convert to fizz buzz covering the two parts of the kata.
* the another one to print the numbers from 1 to 100 as fizz buzz. this should re-use the previous one method.